ios/EnrichedTextInputModule.mm

```diff
@@ -0,0 +1,46 @@
+//
+//  EnrichedTextInputModule.m
+//  ReactNativeEnriched
+//
+//  Created by Ivan Ignathuk on 04/11/2025.
+//
+
+#import "EnrichedTextInputModule.h"
+#import "EnrichedTextInputView.h"
+
+static UIView *findViewByReactTag(NSInteger reactTag, RCTBridge *bridge) {
+  if (bridge.uiManager) {
+    UIView *view = [bridge.uiManager viewForReactTag:@(reactTag)];
+    if (view) {
+      return view;
+    }
+  }
+
+  return nil;
+}
+
+@implementation EnrichedTextInputModule
+
+- (std::shared_ptr<facebook::react::TurboModule>)getTurboModule:(const facebook::react::ObjCTurboModule::InitParams &)params {
+  return std::make_shared<facebook::react::NativeEnrichedTextInputModuleSpecJSI>(params);
+}
+
+- (nonnull NSString *)getHTMLValue:(NSInteger)inputTag {
+  __block NSString *value = @"";
+
+  dispatch_sync(dispatch_get_main_queue(), ^{
```

Comment:
I don't like the fact that we need `dispatch_sync` here. It blocks the current thread.